### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by felipecruz91**
Provides a bug report and a feature request template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+If you are reporting a new issue, make sure that we do not have any duplicates
+already open. You can ensure this by searching the issue list for this
+repository. If there is a duplicate, please close your issue and add a comment
+to the existing issue instead.
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (aevesdocker) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]:
```suggestion
**Add the steps to Reproduce**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (aevesdocker) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]:
```suggestion
**Optional: Add screenshots**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+(paste your output here)
+```
+
+**Output of `docker version`:**
+
+```
+(paste your output here)
+```
+
+**Diagnostics ID**
```
Comment (aevesdocker) [inline .github/ISSUE_TEMPLATE/bug_report.md:45]:
```suggestion
**Include the Diagnostics ID**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ""
+labels: enhancement
+assignees: ""
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (aevesdocker) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
A clear and concise description of what the problem is. For example. 'I'm always frustrated when [...]'
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (felipecruz91) [reply .github/ISSUE_TEMPLATE/bug_report.md:19]: @aevesdocker I'll be using the suggestion below as it contains a lowercase `r` for the `Reproduce` word:

```suggestion
**Add the steps to reproduce**
```

**Review by aevesdocker — APPROVED** [review]
LGTM!

**Comment by felipecruz91** [comment]
Thanks, @aevesdocker! I've pushed your suggestions 😄

**Review by gtardif — APPROVED** [review]

**Pull request merged**